apps/accounts/end_user_signup_forms.py

```diff
@@ -165,9 +165,11 @@ def save(self):
                                    )
 
         # TODO: Add Crosswalk Create.
+        # unhex the salt before use
+        unsalt = binascii.unhexlify(settings.USER_ID_SALT)
         Crosswalk.objects.create(user=new_user,
                                  user_id_hash=binascii.hexlify(pbkdf2(self.cleaned_data['id_number'],
-                                                                      settings.USER_ID_SALT,
+                                                                      unsalt,
```

Comment:
This needs to be in a shared utility function so that we don't make mistake and get these instances out of sync.